test/test_autograd.py

```diff
@@ -1591,6 +1591,15 @@ def test_inplace_view_of_view(self):
         x.sum().backward()
         self.assertEqual(root.grad.data.tolist(), [[1, 2], [1, 1]])
 
+    def test_inplace_view_volatile(self):
+        # an in-place operation on a view that makes the view volatile should
+        # make the base volatile too
+        base = Variable(torch.randn(2, 2))
+        view = base.narrow(0, 0, 1)
+        view.add_(Variable(torch.randn(1, 2), volatile=True))
+        self.assertTrue(view.volatile)
+        self.assertTrue(base.volatile)
```

Comment:
Doing it in the other way should trigger the `requires_grad` consistency checks right? Can you add a case that ensures that this happens?